strapi-community/strapi-plugin-url-alias
# feat(enable button): EditView is hidden if url alias is not enabled

### What does it do?

If a content type has disabled url alias it will no longer show the url alias edit view in the content manager

### Why is it needed?

Less clutter on the page, which gives a better user experience

### How to test it?

Try disabling url alias for one of your content types and check in the content manager that it is longer there

### Related issue(s)/PR(s)

#38
